eng/Packages.Data.props

```diff
@@ -209,7 +209,6 @@
     <PackageReference Update="Microsoft.TypeSpec.Generator.ClientModel" Version="[$(UnbrandedGeneratorVersion)]" />
     <PackageReference Update="Microsoft.TypeSpec.Generator.Input" Version="[$(UnbrandedGeneratorVersion)]" />
     <PackageReference Update="Azure.Generator" Version="[$(AzureGeneratorVersion)]" />
-    <PackageReference Update="System.ClientModel" Version="1.4.1" />
```

Comment:
This matches the global version.  There should be no need for a specific version.